mne_bids/copyfiles.py

```diff
@@ -533,18 +533,19 @@ def copyfile_eeglab(src, dest):
     # structure and potentially breaks re-reading of the file
     uint16_codec = None
     eeg = loadmat(file_name=src, simplify_cells=False,
-                  appendmat=False, uint16_codec=uint16_codec)
+                  appendmat=False, uint16_codec=uint16_codec, mat_dtype=True)
```

Comment:
oh this is a sufficient fix?! How nice would that be

Reply:
Yes, it solved the pb on my end.